react/react1/week2/todo-app/src/components/Timer.js

```diff
@@ -0,0 +1,14 @@
+import { useEffect, useState } from 'react';
+
+const Timer = () => {
+  const [seconds, setSeconds] = useState(0);
+  useEffect(() => {
+    setTimeout(() => {
+      setSeconds((prev) => prev + 1);
+    }, 1000);
+  }, [seconds]);
+
+  return <p>You have spent {Math.floor(seconds)} seconds on this website</p>;
```

Comment:
Why do you need to round your seconds?

Reply:
Leftover from messing around with the counter, I'll remove it 🙈
Thanks a lot for reviewing!